smart-scan/src/test/java/SmartScanTests.java

```diff
@@ -0,0 +1,17 @@
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SmartScanTests {
```

Comment:
Meh, they're not good tests but they're not horrible, I mean you should probably be mocking data rather than relying on an API because that's just how unit tests are supposed to work - what you have here are more related to integration tests.